selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/packages/RenamePackageTest.java

```diff
@@ -363,14 +363,9 @@ public void checkTest3() {
     refactor.setAndWaitStateUpdateReferencesCheckbox(true);
     refactor.setAndWaitStateRenameSubpackagesCheckbox(false);
     refactor.setAndWaitStateUpdateNonJavaFilesCheckbox(false);
-    loader.waitOnClosed();
-    refactor.clickOkButtonRefactorForm();
-    loader.waitOnClosed();
-    refactor.waitRenamePackageFormIsClosed();
-    projectExplorer.waitItemInvisibility(PROJECT_NAME + "/src/main/java/test3/fred");
-    projectExplorer.waitItem(PROJECT_NAME + "/src/main/java/test3/r");
-    editor.waitTextIntoEditor(TEST3_R_OUT);
-    editor.closeFileByNameWithSaving("A");
+    refactor.waitTextInErrorMessage("Package already exists");
```

Comment:
If I understand it correctly, this rename used to work, but does not anymore. Is this the behvavior we get from Eclipse refactoring?

Reply:
see my comment above